### PR TITLE
Added Croatian translation (hr_HR)

### PR DESCRIPTION
**Pull request by gseric**

### CODE REVIEW

**Comment by jacquesbh** [comment]
Hey @gseric ! Thank you for that!

Do you think you can also translate the v2 of this project? It's way better than the v1 and should be released soon, it's already been in RC for a moment.

**Comment by gseric** [comment]
No problem, please show me what I missed. I translated from english version on master, isn't it latest 2.0 RC version already?

**Comment by jacquesbh** [comment]
> No problem, please show me what I missed. I translated from english version on master, isn't it latest 2.0 RC version already?

Actually yes it is! My bad :p.

I'll do a quick review and merge tonight.

File: src/Resources/translations/messages.hr.yml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      short_description: 'Naslov razine 2.'
+      description: 'Element koji predstavlja naslov druge razine (podnaslov).'
+      field:
+        content: 'Sadržaj'
+    monsieurbiz.h3:
+        title: '<h3> naslov'
+        short_description: 'Naslov razine 3.'
+        description: 'Element koji predstavlja naslov treće razine (pod-podnaslov).'
+      field:
+        content: 'Sadržaj'
```
Comment (jacquesbh) [inline src/Resources/translations/messages.hr.yml:36]:
```suggestion
    monsieurbiz.h3:
      title: '<h3> naslov'
      short_description: 'Naslov razine 3.'
      description: 'Element koji predstavlja naslov treće razine (pod-podnaslov).'
      field:
        content: 'Sadržaj'
```

**Review by jacquesbh — CHANGES_REQUESTED** [review]
Just a small fix :).

File: src/Resources/translations/messages.hr.yml (hunk 1 above)
Comment (gseric) [reply src/Resources/translations/messages.hr.yml]: Nice catch :)

**Review by jacquesbh — APPROVED** [review]

File: src/Resources/translations/messages.hr.yml (hunk 1 above)
Comment (jacquesbh) [reply src/Resources/translations/messages.hr.yml]: I just ran the plugin and tried it :p

**Comment by jacquesbh** [comment]
Thank you @gseric !! 🇭🇷 🎉

**Pull request merged**